README.adoc

```diff
@@ -43,7 +43,7 @@ include::finish/src/resources/artists.json[]
 ----
 
 You will implement an AngularJS client that consumes this JSON and displays its contents at the
-following URL: http://localhost:9080[http://localhost:9080^].
+following URL: \http://localhost:9080/index.html.
```

Comment:
Is the forward slash at the start of the URL needed?

Reply:
Link should be in `backticks`

Reply:
I don't want reader to run the url at there because server is not installed and started